testing/TestHarness/TestHarness.UITest/AuthHttpUtilityTests.cs

```diff
@@ -6,10 +6,11 @@ namespace TestHarness.UITest;
 public class AuthHttpUtilityTests 
 {
 	[Test]
+	[Ignore("Removed to avoid direct dependency on Uno.Extensions libraries")]
```

Comment:
Should there be an issue to fix it? Is is possible?

Reply:
See https://github.com/unoplatform/uno.extensions/issues/685